GDJS/GDJS/IDE/Exporter.cpp

```diff
@@ -133,14 +134,26 @@ bool Exporter::ExportWholePixiProject(const ExportOptions &options) {
       return false;
     }
 
+    auto projectUsedResources =
+        gd::UsedResourcesFinder::FindProjectUsedResources(exportedProject);
+    std::vector<std::set<gd::String>> layersUsedResources;
```

Comment:
```suggestion
    std::vector<std::set<gd::String>> layoutsUsedResources;
```
or scenesUsedResources